tests/migrations/test_autodetector.py

```diff
@@ -1350,6 +1350,29 @@ def test_add_date_fields_with_auto_now_add_not_asking_for_null_addition(
         self.assertOperationFieldAttributes(changes, "testapp", 0, 1, auto_now_add=True)
         self.assertOperationFieldAttributes(changes, "testapp", 0, 2, auto_now_add=True)
 
+    def test_remove_primary_key_attribute(self):
+        initial_state = ModelState(
+            "testapp",
+            "Author",
+            [
+                ("pkfield", models.IntegerField(primary_key=True)),
+            ],
+        )
+        updated_state = ModelState(
+            "testapp",
+            "Author",
+            [
+                ("id", models.BigAutoField()),
```

Comment:
This should be a primary key.
```suggestion
                ("id", models.BigAutoField(primary_key=True)),
```